src/hooks-worker-client.coffee

```diff
@@ -9,23 +9,19 @@ logger = require './logger'
 which = require './which'
 
 
-HOOK_TIMEOUT = 5000
-
-CONNECT_TIMEOUT = 1500
-CONNECT_RETRY = 500
-AFTER_CONNECT_WAIT = 100
-
-TERM_TIMEOUT = 5000
-TERM_RETRY = 500
-
-HANDLER_HOST = 'localhost'
-HANDLER_PORT = 61321
-HANDLER_MESSAGE_DELIMITER = "\n"
-
-
 class HooksWorkerClient
   constructor: (@runner) ->
-    @language = @runner.hooks?.configuration?.options?.language
+    options = @runner.hooks.configuration.options
```

Comment:
@netmilk I tried to remove `runner.hooks.configuration`, but it seems it's there on purpose and it's part of the Node.js API: http://dredd.readthedocs.org/en/latest/hooks-nodejs/#api-reference